dm/pkg/binlog/pos_finder.go

```diff
@@ -0,0 +1,369 @@
+// Copyright 2019 PingCAP, Inc.
```

Comment:
please fix the copyright and other files, but i am wonder why `make check-copyright`don't work 😂

Reply:
only checks its existence
https://github.com/pingcap/tiflow/blob/abefe57c154efb28f91cf1a15702988e1a7249e7/scripts/check-copyright.sh#L4